app/controllers/api/v3x1/bulk_create_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V3x1
+    class BulkCreateController < ApplicationController
+      def create
+        # Authorize creating a new source, before we go through the processing.
+        authorize(Source.new)
```

Comment:
@lindgrenj6 When we have an intermediary (e.g Cloud Controller) creating the Source object should it be building a temporary XRH with the account_number in it when it calls the end point? Do you support Pre Shared Keys (PSK) that can  be used to trust connections from services running inside the cloud.redhat.com.

Reply:
We could support a PSK - but the x-rh-id is more for the account number than anything. 

If we wanted to do an internal bulk-create operation I could see us also offering a kafka consumer for that operation. Not sure if that's in the cards right now though, but we could do it eventually.